### PR TITLE
Use field type constants

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: src/Og.php [hunk 1]
```diff
@@ -413,12 +413,10 @@ public static function getGroupCount(EntityInterface $entity, $group_type_id = N
...
     $group_content = [];
 
+
     // Retrieve the fields which reference our entity type and bundle.
     $query = \Drupal::entityQuery('field_storage_config')
-      // @todo For the moment retrieving both group types, since there seems to
-      //   be some confusion about which field type is used for users.
-      // @see https://github.com/amitaibu/og/issues/177
-      ->condition('type', ['og_standard_reference', 'og_membership_reference'], 'IN');
+      ->condition('type', OgGroupAudienceHelper::NON_USER_REFERENCE_FIELD);
```
Comment (amitaibu) [inline src/Og.php]: as we now separate user and non-user logic, the query here should only bring non-user entities.

**Comment by amitaibu** [comment]
@pfrenssen ready for review

**Comment by pfrenssen** [comment]
Constant names are still a bit weird but I cannot come up with anything clearer.

Maybe @damiankloip can give a good suggestion as a native speaker?

**Comment by amitaibu** [comment]
Lets get this in, and revisit if needed.

**Pull request merged**